portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -41,18 +43,29 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    saveComment(request.getParameter("display-name"), request.getParameter("comment"));
-    response.sendRedirect("/aboutme.html");
+    if (userService.isUserLoggedIn()) {
+      String userEmail = userService.getCurrentUser().getEmail();
+      String urlToRedirectToAfterUserLogsOut = "/";
```

Comment:
This can be class variable
private static final String LOGOUT_REDIRECT_URL = "/";